src/views/topics/TopicsListView.vue

```diff
@@ -0,0 +1,159 @@
+<script setup>
```

Comment:
The search is not working for `defis` (points to datasets). 

It should probably be disabled in configuration. And many things from this View can be removed since they're related to `query`.

Or I guess the search can be configured to point to topics.

Reply:
i disabled it with a config